src/services/pokemon.js

```diff
@@ -36,27 +36,27 @@ export const fetchPokemon = async () => {
   return pokemonList;
 };
 
-export const fetchSearchPokemon = (pokemonName) => {
+export const fetchSearchPokemon = async (pokemonName) => {
   const lowerCaseName = pokemonName.toLowerCase();
-  return fetch(
+  return await fetch(
```

Comment:
because this fetch call is using .then it does not need await in front of it. The .then makes sure what happened before completes its action before moving forward.